Cargo.toml

```diff
@@ -26,7 +26,7 @@ cargo-util = { path = "crates/cargo-util", version = "0.1.1" }
 crates-io = { path = "crates/crates-io", version = "0.33.0" }
 crossbeam-utils = "0.8"
 curl = { version = "0.4.38", features = ["http2"] }
-curl-sys = "0.4.47"
+curl-sys = "0.4.45"
```

Comment:
Since Cargo defaults to a caret dependency, will this continue to pull the broken version?

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Or will the broken version be yanked, so this is okay?

(You're probably on top of this; just want to make sure 😅)

Reply:
I manually downgraded the lockfile in rust-lang/rust (see https://github.com/rust-lang/rust/pull/89079).  Will update to 0.4.48 when a new version of libcurl is published next week.